lib/grape/jsonapi/entity/resource.rb

```diff
@@ -40,6 +64,7 @@ def self.attribute(*args, &block)
         end
 
         def self.nest(name, options = {})
+          return if options[:using] && avoid_recursive_relation_check(options[:using])
           _expose_relationships(name, options)
```

Comment:
maybe always expose the relationship and just skip the include?

Reply:
Done